README.md

```diff
@@ -103,15 +104,15 @@ A super awesome collection of regularly used custom hooks as utils for React.
 Import any hook from "rooks" and start using them!
 
 ```jsx
-import { useDidMount } from "rooks";
+import { useDidMount } from 'rooks';
 ```
 
 ## Usage
 
 ```jsx
 function App() {
   useDidMount(() => {
-    alert("mounted");
+    alert('mounted');
```

Comment:
@iampika Do you have some formatted enabled on your VSCode? Looks like we are getting unnecessary changes in the PR.

Reply:
fixed